polaris-react/src/components/Modal/components/CloseButton/CloseButton.tsx

```diff
@@ -19,8 +21,18 @@ export function CloseButton({
   onClick,
 }: CloseButtonProps) {
   const i18n = useI18n();
+  const {polarisSummerEditions2023} = useFeatures();
 
-  return (
+  return polarisSummerEditions2023 ? (
+    <Button
+      primary
+      plain
+      pressed={pressed}
+      icon={CancelMajor}
+      onClick={onClick}
+      accessibilityLabel={i18n.translate('Polaris.Common.close')}
+    />
+  ) : (
```

Comment:
Our buttons are good. Why not use them? 🤔 🤷

Reply:
With rendering the render Button with the se flag on, there's se specific code that was added to `CloseButton.scss` that can be cleaned up 👍

Reply:
Great catch - have cleaned them up 👍